### DISCUSSION TITLE
[IntelliSense] Slightly complex classRegex not working

### DISCUSSION THREAD

**Discussion by heyjoe1984** [tailwindlabs/tailwindcss#6456] [Help]
I'm trying to use this classRegex:

    "tailwindCSS.experimental.classRegex": [
        "(?:\\G(?!\\A)',\\s*'|class\\(\\[\\s*')\\K[^']*",
    ],

As you can see, it works here: https://regex101.com/r/es1U5W/1

However it doesn't seem to be working in VS Code.

**Comment by bradlc** [comment]
Hey @heyjoe1984. There are definitely some edge cases that don't work with the `experimental.classRegex` setting currently, but in this case the first thing to check is that your regular expression works in JavaScript. If you select `ECMAScript (JavaScript)` on regex101 you can see that your regular expression does not match your test string: https://regex101.com/r/8QaGUD/1
